### PR TITLE
Resources taxonomy

### PR DESCRIPTION
**Pull request by IvarBorthen**
Merge this first: https://github.com/NDLANO/frontend-packages/pull/346
• Taxonomy for a resource
• Same props in and same data out as before

### CODE REVIEW

File: src/containers/LearningResourcePage/components/LearningResourceForm.jsx [hunk 1]
```diff
@@ -310,7 +312,11 @@ class LearningResourceForm extends Component {
         ),
       },
     ];
-    if (model.id) {
+
+    if (
+      model.id &&
+      userAccess.includes(`taxonomy-${config.ndlaEnvironment}:write`)
```
Comment (gunnarvelle) [inline src/containers/LearningResourcePage/components/LearningResourceForm.jsx]: Denne mister snart miljø-delen av verdien. Slik at det blir taxonomy:write. Er lurt om det kan tas høgde for også.

File: src/containers/LearningResourcePage/components/LearningResourceTaxonomy.jsx [hunk 2]
```diff
@@ -9,133 +9,466 @@
...
+
+  async getResourceId() {
+    const { articleId, language } = this.props;
+    let resourceId = '';
+    const resource = await queryResources(articleId, language);
+    if (resource.length > 0) resourceId = resource[0].id;
+    return resourceId;
+  }
+
+  async fetchTaxonomy() {
```
Comment (oyvinmar) [inline src/containers/LearningResourcePage/components/LearningResourceTaxonomy.jsx]: Maybe move all fetching logic to taxonomyApi? (i.e. export fetchTaxonomy and fetchTaxonomyChoices)

File: src/containers/LearningResourcePage/components/LearningResourceTaxonomy.jsx [hunk 3]
```diff
@@ -9,133 +9,466 @@
...
+        ...topic,
+        primary: topic.id === id,
+      })),
+    });
+  }
+
+  async getResourceId() {
+    const { articleId, language } = this.props;
+    let resourceId = '';
+    const resource = await queryResources(articleId, language);
```
Comment (oyvinmar) [inline src/containers/LearningResourcePage/components/LearningResourceTaxonomy.jsx]: Do we handle cases where an article is used in several resources?

File: src/containers/LearningResourcePage/components/LearningResourceTaxonomy.jsx [hunk 4]
```diff
@@ -9,133 +9,466 @@
...
+      } else {
+        this.setState(prevState => ({
+          resourceTaxonomy: {
+            ...prevState.resourceTaxonomy,
+            isLoadingResource: false,
+          },
+        }));
+      }
+    } catch (e) {
+      handleError(e);
```
Comment (oyvinmar) [inline src/containers/LearningResourcePage/components/LearningResourceTaxonomy.jsx]: We should give the user some feedback if something fails. Maybe isLoadingResouces and isLoadingTaxonomy can be renamed to `status` which can have the following possible values `'initial' | 'loading' | 'success' | 'error'`. Each value should be handled in render.

**Pull request merged**